packages/manager/modules/pci/src/projects/project/storages/databases/components/user-roles/user-roles.controller.js

```diff
@@ -0,0 +1,10 @@
+export default class {
+  $onInit() {
+    this.selectedRoles = [];
+  }
+
+  submit() {
+    this.isLoading = true;
```

Comment:
is isLoading used ? it's never set to false

Reply:
It is not set to false because the modal has to close after the change is done. But it does not close immediately, because we might reload the destination page and this modal remains open until the loading is complete. Setting this flag to false makes the modal buttons enabled.